cmd/util/feature_flags.go

```diff
@@ -30,6 +30,7 @@ const (
 	NODE_OP            FeatureFlag = "NODE_OPS"
 	TOOLS              FeatureFlag = "TOOLS"
 	AZURE_CIDR_ALLOWED FeatureFlag = "AZURE_CIDR_ALLOWED"
+	CLUSTER_CMK_UPDATE FeatureFlag = "CLUSTER_CMK_UPDATE"
```

Comment:
Do we have YBM CLI UTs to test this?

Reply:
@kc-yb - what unit tests do you have in mind? Is it for this specific feature flag or the feature flag functionality in general?

Reply:
In general, and for this as well - ie, turning the feature flag does not show the command

Reply:
If you have doc on YBM CLI UT appreciate if you can share it